OmiseSDK/AtomeFormViewModel.swift

```diff
@@ -0,0 +1,114 @@
+    var fields: [Field] = [
+        .name,
+        .email,
+        .phoneNumber,
+        .street1,
+        .street2,
+        .city,
+        .state,
+        .country,
+        .postalCode
+    ]
+
+    // MARK: CountryListViewModelProtocol
+    lazy var countries: [CountryInfo] = CountryInfo.all.sorted { $0.name < $1.name }
+    lazy var selectedCountry: CountryInfo? = CountryInfo.current {
+        didSet {
+            if let selectedCountry = selectedCountry {
+                onSelectCountry(selectedCountry)
+            }
+        }
+    }
+    var onSelectCountry: (CountryInfo) -> Void = { _ in }
+
+    var countryListViewModel: CountryListViewModelProtocol { return self }
+    
+    var fieldForShippingAddressHeader: Field? { .street1 }
+    var submitButtonTitle = "Atome.submitButton.title".localized()
+    var headerText = "Atome.header.text".localized()
+    var logoName = "Atome_Big"
+
+    private let flowSession: PaymentCreatorFlowSession?
+
+    init(flowSession: PaymentCreatorFlowSession?) {
+        self.flowSession = flowSession
+    }
+
+    func title(for field: Field) -> String? {
+        var title = field.title.localized()
+        if field.isOptional {
+            title += " " + "Atome.field.optional".localized()
+        }
+        return title
+    }
+
+    func error(for field: Field, validate text: String?) -> String? {
+        if field.isOptional, text?.isEmpty ?? true { return nil }
+
+        if let validatorRegex = field.validatorRegex,
+            let regex = try? NSRegularExpression(pattern: validatorRegex, options: []) {
+            do {
+                try regex.validate(text ?? "")
+                return nil
+            } catch {
+                return field.error ?? error.localizedDescription
+            }
+        } else {
+            if let text = text, text.isEmpty {
+                return field.error
+            }
+        }
+
+        return nil
+    }
+
+    func onSubmitButtonPressed(_ viewContext: ViewContext, onComplete: @escaping () -> Void) {
+        guard let flowSession = flowSession else {
+            onComplete()
+            return
+        }
+
+        let shippingAddress: PaymentInformation.Atome.ShippingAddress =
+            .init(country: viewContext[.country],
+                  city: viewContext[.city],
+                  postalCode: viewContext[.postalCode],
+                  state: viewContext[.state],
+                  street1: viewContext[.street1],
+                  street2: viewContext[.street2])
+
+        let items: [PaymentInformation.Atome.Item] = [
+            .init(
+                sku: "3427842",
+                category: "Shoes",
+                name: "Prada shoes",
+                quantity: 1,
+                amount: flowSession.paymentAmount ?? 0,
+                itemUri: "omise.co/product/shoes",
+                imageUri: "omise.co/product/shoes/image",
+                brand: "Gucci"
+            )
+        ]
```

Comment:
Is it test data?

Reply:
It's a mock up data as Atome requires Item object while other payment methods don't. So right now it's a work around that will need to be discussed later. This Item object I took from Android app but replaces `www.ken.com` with `omise.co`

Reply:
It cannot be empty array, right?